lib/matplotlib/axes/_axes.py

```diff
@@ -5233,6 +5268,11 @@ def get_interp_point(ind):
         collection = mcoll.PolyCollection(polys, **kwargs)
 
         # now update the datalim and autoscale
+        # For between pad last value
+        if step == 'between':
+            y1 = np.r_[y1, y1[-1]]
+            y2 = np.r_[y2, y2[-1]]
```

Comment:
`np._r` is obscure enough that I had to look it up.  Is there some reason to use this instead of `np.array`?

Reply:
It's just a convenient way of appending to arrays. Could you use bunch of other things, but they are more verbose. It is already used in a few places in mpl like `contour.py`